documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -13,6 +13,7 @@
 ### New features
 
 * npm lockfile and shrinkwrap detectors now ignore packages flagged as extraneous in the package-lock.json and npm-shrinkwrap.json files.
+* Support added for Opam Package Manager via [Opam Detectors](https://opam.ocaml.org/).
```

Comment:
Suggest linking to the Opam page (opam.md) here in the release notes and then on the Opam page itself, add the Opam Detectors link as a reference.

Reply:
Done.